README.md

```diff
@@ -1,8 +1,7 @@
 # .NET API Reference Docs
 
-This repo contains documentation for the .NET API Reference.
+This repo contains documentation for the .NET API reference. We track all work for this repository using [GitHub issues](https://github.com/dotnet/dotnet-api-docs/issues). The documentation for APIs is built from the text and code snippets in this repo, and from the samples in the [dotnet/samples](https://github.com/dotnet/samples) repository.
```

Comment:
Would this be a reasonable location to add a comment about regarding the move to use `///` comments in the runtime repo as the source of truth, over time? I'm not sure. It could be useful, but details on which namespaces have been converted could quickly become out of date.

Reply:
That's a good point, but I think we should only add that note once we've fully made the switch - so as not to confuse people.